### PR TITLE
Show how SVG is basically the same

### PR DESCRIPTION
**Pull request by annevk**

### CODE REVIEW

**Comment by annevk** [comment]
Created for @erikdahlstrom.

File: element-constructors.js [hunk 1]
```diff
@@ -162,6 +164,20 @@ class CustomElement extends HTMLElement {
   ...
 }
 
+// SVGElement is identical to HTMLElement except it does not have lowercasing
+class SVGElement extends Element {
+  constructor({ localName = undefined, prefix = undefined, document = undefined } = {}) {
+    super({ localName, namespace: HTML_NS, prefix, document });
```
Comment (domenic) [inline element-constructors.js]: SVG_NS presumably

Comment (annevk) [inline element-constructors.js]: Sorry about that

**Comment by domenic** [comment]
Yay! Merged as cdfe5a1d865e25d265074418df7918fda959e403

**Pull request closed without merging**